src/decrypter/ntoh.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Convert network-order (big-endian) bytes into their little-endian
+ * representation.
+ */
+const ntoh = function(word) {
```

Comment:
Personally, I think breaking out single functions that aren't used outside a particular module is overkill and makes things harder to track down.